src/meta-srv/src/version_reporter.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use common_meta::rpc::store::{PutRequest, RangeRequest};
+use common_meta::version_reporter::{
+    GreptimeVersionReport, Mode as VersionReporterMode, Reporter, VersionReportTask,
+    VERSION_REPORT_INTERVAL, VERSION_UUID_KEY,
+};
+
+use crate::cluster::MetaPeerClientRef;
+use crate::service::store::kv::KvStoreRef;
+
+struct MetaVersionReport {
+    meta_peer_client: MetaPeerClientRef,
+    kv_store: KvStoreRef,
+    uuid: Option<String>,
+    retry: i32,
+    uuid_key_name: Vec<u8>,
+}
+
+fn get_uuid_key_name() -> Vec<u8> {
```

Comment:
question: did I get it right that this has to be a method rather then a constant (something like `&[u8]`) because the const cannot refer to static?

Reply:
I will fix it.